tasks/Task.cpp

```diff
@@ -110,13 +119,9 @@ bool Task::handleIncomingWebsocketMessage(char const* data, WebSocket *connectio
     if (is_controlling) {
         return handleControlMessage();
     }
-
-    if (!handleAskControlMessage()){
-        connection->close();
-        return false;
+    else {
+        return handleAskControlMessage();
```

Comment:
Why did you change your mind about closing the connection?

Also, don't need to use else.

Reply:
See the message from https://github.com/rock-drivers/drivers-orogen-controldev_websocket/pull/2/commits/aca4d7b23b55f4a4c840c2fdf8a2a886e9ae7f7b

Reply:
The if / else in this very particular instance ... is a matter of taste. 

In this case, I like it because it shows the symmetry. I would not have commented on it before, because it *really* is a matter of taste.